README.md

```diff
@@ -7,122 +7,86 @@
-from griptape.drivers import (
-    GriptapeCloudEventListenerDriver,
-    OpenAiChatPromptDriver,
-)
-from griptape.events import (
-    EventListener,
-)
-from griptape.structures import Agent
-from griptape.tools import Calculator
-
-structure = Agent(
-    tools=[Calculator(off_prompt=False)],
-    config=StructureConfig(
-        global_drivers=StructureGlobalDriversConfig(
-            prompt_driver=OpenAiChatPromptDriver(
-                model="gpt-4",
-                stream=True,
-            ),
-        )
-    ),
-    event_listeners=[
-        EventListener(
-            driver=GriptapeCloudEventListenerDriver(
-                base_url="http://127.0.0.1:5000",
-                api_key=os.environ["GRIPTAPE_CLOUD_API_KEY"],
-            ),
-        )
-    ],
-)
-
-structure.run(sys.argv[1]) # Structure Run arguments are passed in via standard input. 
-```
+    ```bash
+    cd managed-structure-template
+    ```
+3. Start the emulator.
+   ```bash
+   gt server start
+   ```
+4. In a separate terminal window, register a Structure.
+   ```bash
+   gt server register --directory . --main-file structure.py
+   ```
 
-And a `requirements.txt` file with the following content.
-```
-griptape
-```
+   The example client program uses the environment variable `GT_STRUCTURE_ID` to determine which Structure to run.
+   Set this environment variable to the Structure ID you registered in the previous step.
+   ```bash
+   export GT_STRUCTURE_ID={STRUCTURE_ID}
+   ```
 
-3. Start the emulator.
-```
-gt cloud server start
-```
+   Or you can register and set the environment variable in one step.
+   ```bash
+   export GT_STRUCTURE_ID=$(gt server register --directory . --main-file structure.py)
+   ```
 
-By default, the emulator runs on `127.0.0.1` and port `5000`. You can change these values by providing the `--host` and `--port` options.
-```
-gt cloud server start --host localhost --port 5001
-```
-We'll continue with the default values for the rest of this guide.
+> [!IMPORTANT]
+> Structures registered with the emulator are not persisted across restarts. You will need to re-register the Structure each time you restart the emulator.
+5. Confirm that the Structure is registered.
+   ```bash
+   gt server list
+   ```
+6. Add your OPENAI_API_KEY to the `.env` file.
+   ```bash
+   echo "OPENAI_API_KEY=sk-XXXXXX-XXXXXX-XXXXXX-XXXXXX" >> .env
+   ```
+7. Rebuild the structure to load in the new environment variable. 
+   Note that this is only required for changes to `.env` or `requirements.txt`. Code changes do not require a rebuild. 
+   ```bash
+   gt server build
+   ```
+8. Now that your Structure is registered, use the example client program to call the emulator's API for running the Structure.
 
-4. In a separate terminal window, register a Structure.
-```
-gt cloud server register --directory skatepark-local --main-file structure.py
-```
+   Install the dependencies required by the example client program.
```

Comment:
is poetry a requirement?

Reply:
Only for the example client program. Perhaps we need a better repo structure to clearly highlight the differences between the structure and the example client program.